contrib/pkg_tester/__snapshots__/test_fdb_pkgs.ambr

```diff
@@ -68,24 +68,24 @@
                    the CLI.
     --api-version  APIVERSION
                    Specifies the version of the API for the CLI to use.
-    --tls_certificate_file CERTFILE
+    --tls-certificate_file CERTFILE
```

Comment:
```suggestion
    --tls-certificate-file CERTFILE
```

Reply:
Good catch, my grep did not pick up options with a hyphen before an underscore, so I missed these three. I ran it again and didn't see any others that I missed.